apps/prairielearn/src/pages/instructorAssessmentQuestions/instructorAssessmentQuestions.html.ts

```diff
@@ -0,0 +1,399 @@
+                  </h4>
+                </div>
+                <div class="modal-body">
+                  <p>
+                    Are your sure you want to reset all current variants of this question?
+                    <strong>All ungraded attempts will be lost.</strong>
+                  </p>
+                  <p>Students will receive a new variant the next time they view this question.</p>
+                </div>
+                <div class="modal-footer">
+                  <form name="reset-question-variants-form" method="POST">
+                    <input type="hidden" name="__action" value="reset_question_variants" />
+                    <input type="hidden" name="__csrf_token" value="${resLocals.__csrf_token}" />
+                    <input
+                      type="hidden"
+                      name="unsafe_assessment_question_id"
+                      class="js-assessment-question-id"
+                      value=""
+                    />
+                    <button type="button" class="btn btn-secondary" data-dismiss="modal">
+                      Cancel
+                    </button>
+                    <button type="submit" class="btn btn-danger">Reset question variants</button>
+                  </form>
+                </div>
+              </div>
+            </div>
+          </div>
+
+          ${renderEjs(
+            __filename,
+            "<%- include('../partials/assessmentSyncErrorsAndWarnings'); %>",
+            resLocals,
+          )}
+
+          <div class="card mb-4">
+            <div class="card-header bg-primary text-white d-flex align-items-center">
+              ${resLocals.assessment_set.name} ${resLocals.assessment.number}: Questions
+            </div>
+            ${AssessmentQuestionsTable({
+              questions,
+              assessmentType: resLocals.assessment.type,
+              urlPrefix: resLocals.urlPrefix,
+              hasCourseInstancePermissionEdit:
+                resLocals.authz_data.has_course_instance_permission_edit,
+            })}
+          </div>
+        </main>
+      </body>
+    </html>
+  `.toString();
+}
+
+function AssessmentQuestionsTable({
+  questions,
+  urlPrefix,
+  assessmentType,
+  hasCourseInstancePermissionEdit,
+}: {
+  questions: AssessmentQuestionRow[];
+  assessmentType: string;
+  urlPrefix: string;
+  hasCourseInstancePermissionEdit: boolean;
+}) {
+  let nTableCols = 11;
+
+  // If at least one question has a nonzero unlock score, display the Advance Score column
+  const showAdvanceScorePercCol =
+    questions.filter((q) => {
+      return q.assessment_question_advance_score_perc !== 0;
+    }).length >= 1;
+  if (showAdvanceScorePercCol) nTableCols++;
+
+  function maxPoints({ max_auto_points, max_manual_points, points_list, init_points }) {
+    if (max_auto_points || !max_manual_points) {
+      if (assessmentType === 'Exam') {
+        return `${(points_list || [max_manual_points]).map((p) => p - max_manual_points)}`;
```

Comment:
Will this render the points how we want them to? Currently on master it's something like `1,2,3,4`, but I think that only worked because of the way that EJS renders lists by default. I worry that we'll end up rendering `[1,2,3,4]` or `1234`. Can you check this with a question with an array of points?

Reply:
Because of the string interpolation here, the commas get added in when rendering. 
![Screenshot 2024-04-09 at 11 00 46 AM](https://github.com/PrairieLearn/PrairieLearn/assets/114321556/8910339a-75c1-4405-8737-bda1b83bc1f2)

Reply:
Can we change this to be explicit about how this is rendered? That is, rather than relying on the fact that the array is implicitly converted to a string, let's do something like this instead:

```ts
return (points_list || [max_manual_points]).map(p => p - max_manual_points).join(',');
```